### PR TITLE
Fix verify email functionality

### PR DESCRIPTION
**Pull request by mjcalvo21**
https://trello.com/c/Fjjk3J6n/4566-cant-add-or-delete-emails-on-a-record

https://trello.com/c/0aNqhdbo/4567-email-frequency-doesnt-save -> Requires backend review

### CODE REVIEW

**Comment by amontenegro** [comment]
Im reviewing and fixing a 500 error on verify

**Comment by amontenegro** [comment]
[Email frequency doesn't save](https://trello.com/c/0aNqhdbo/4567-email-frequency-doesnt-save)

File: orcid-web/src/main/webapp/static/javascript/ng1Orcid/app/shared/preferences.service.ts [hunk 1]
```diff
@@ -41,8 +41,7 @@ export class PreferencesService {
     }
 
     updateEmailFrequency( obj ): Observable<any> {
-        let encoded_data = JSON.stringify( obj ); //this.prefs['email_frequency']
-        
+    	let encoded_data = obj;
```
Comment (rcpeters) [inline orcid-web/src/main/webapp/static/javascript/ng1Orcid/app/shared/preferences.service.ts]: Not encoded?

File: orcid-web/src/main/resources/freemarker/common/.fuse_hidden00001af10000fdac [hunk 2]
```diff
@@ -0,0 +1,199 @@
+<#--
```
Comment (rcpeters) [inline orcid-web/src/main/resources/freemarker/common/.fuse_hidden00001af10000fdac]: hidden file?

Comment (amontenegro) [reply orcid-web/src/main/resources/freemarker/common/.fuse_hidden00001af10000fdac]: It looks it is a hidden file, I just removed it

File: orcid-web/src/main/webapp/static/javascript/ng1Orcid/app/shared/preferences.service.ts (hunk 1 above)
Comment (amontenegro) [reply orcid-web/src/main/webapp/static/javascript/ng1Orcid/app/shared/preferences.service.ts]: I encoded it in the latest commit

**Pull request merged**